app/models/post.rb

```diff
@@ -1,2 +1,13 @@
 class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+
+  after_save :update_user_posts_count
+
+  def update_user_posts_count
```

Comment:
[OPTIONAL] I would suggest putting the `update_user_posts_count` `method` after the private method.